gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Jan 30 18:27:36 CET 2017
+#Sat May 02 19:00:33 CEST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```

Comment:
I can hardly believe that we jump more than 6 (?) version without major issues? From my experience there is always a lot of stuff broken. Did you really test this with the new wrapper? (Especially since you removed the wrapper task.)

Reply:
Yes and removing the wrapper task was one thing that I needed to do to get it to work...

Reply:
We have the latest Gradle running with wrapper task at work. This is suspicious. Makes me think that then Travis just uses an older Gradle version.

Reply:
???

Reply:
Well instead of overwritting you can edit it yes I can do that instead

Reply:
Look at @micheljung PR he does the same :D

Reply:
I was surprised, too, but I executed some Gradle tasks and there were no problems :)